crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -127,157 +123,326 @@ pub(crate) enum RelayChain {
 	PolkadotLocal,
 }
 
+/// Command for generating a chain specification.
 #[derive(Args)]
 pub struct BuildSpecCommand {
 	/// File name for the resulting spec. If a path is given,
 	/// the necessary directories will be created
-	/// [default: ./chain-spec.json].
 	#[arg(short = 'o', long = "output")]
 	pub(crate) output_file: Option<PathBuf>,
 	/// This command builds the node if it has not been built already.
 	/// For production, always build in release mode to exclude debug features.
-	#[clap(short = 'r', long, default_value = "true")]
+	#[arg(short = 'r', long)]
 	pub(crate) release: bool,
 	/// Parachain ID to be used when generating the chain spec files.
-	#[arg(short = 'i', long = "id")]
+	#[arg(short = 'i', long)]
 	pub(crate) id: Option<u32>,
 	/// Whether to keep localhost as a bootnode.
-	#[clap(long)]
+	#[arg(long)]
```

Comment:
For context to the rest of reviewers of this change: https://github.com/r0gue-io/pop-cli/issues/353